doc/source/whatsnew/v1.3.0.rst

```diff
@@ -458,6 +458,36 @@ with a :class:`MultiIndex` in the result. This can lead to a perceived duplicati
     df.groupby('label1').rolling(1).sum()
 
 
+.. _whatsnew_130.notable_bug_fixes.quarterbegin_default_anchor:
```

Comment:
you need to say deprecation about the `startingMonth` as its mentioned below as well. I would prefer making these 2 distinct and unrelated changes (which they are). so don't mention deprecation at all here.